examples/tagging/tagging_trainer.py

```diff
@@ -15,7 +15,6 @@
 from typing import Iterator, Dict
 
 import torch
-from texar.torch.data import Batch
```

Comment:
For examples, you should should tell the users to install texar instead of throwing the exception during excecution